mne/realtime/lsl_client.py

```diff
@@ -23,7 +23,10 @@ class LSLClient(_BaseClient):
         the channel type is EEG, the `standard_1005` montage is used for
         electrode location.
     host : str
-        The LSL identifier of the server.
+        The LSL identifier of the server. This is the source_id designated
```

Comment:
How is someone supposed to find out what this source_id is? As I understand, it's not the only way to create the string. You can also use a name. In that case, how do you get the source_id?

Reply:
Name is the same thing as `source_id`, that’s the LSL terminology. It should be expected that there is some familiarity with the pylsl library to use the client. I reduced the complexity of all the different ways one might resolve a LSL stream. The name/source_id, should be a unique identifier that is given at the initiation of the stream. That way there will only be one stream associated with the resolution. If you were to do it by device type, you might have multiple EEGs streaming and you wouldn’t be able to disambiguate which one.

Reply:
In pylsl.resolve_byprop for example, the "name" and "source_id" keywords are distinct. The source_id is unique and the name is not unique.

Reply:
With Cognionics hardware for example, the Cognionics Acquisition Software generates an LSL stream but does not create a source_id for the stream, it just creates a name that's taken from the serial number. In this case I have to use the "name" keyword (which indeed is not unique and could be a problem for multiple devices).